src/devices/Mcp3008/samples/Mcp3008.Sample.cs

```diff
@@ -33,27 +33,27 @@ Mcp3008 GetMcp3008WithSpi()
                 };
 
                 var spi = new UnixSpiDevice(connection);
-                var mcp3008 = new Mcp3008(spi);
+                var mcp3008 = new Mcp3008.Mcp3008(spi);
                 return mcp3008;
             }
 
             // GPIO (via bit banging) implementation
```

Comment:
just a note on this comment, isn't this using HW SPI support underneath (not software SPI, ie bit banging)